src/lib/utils.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import crypto from 'crypto';
+import { promisify } from './promisify';
+
+export const generateRandomToken = async (): Promise<string> => {
```

Comment:
it's exact how oauth2-server generates its tokens: https://github.com/oauthjs/node-oauth2-server/blob/master/lib/utils/token-util.js#L20.